sdk/openai/openai/test/public/node/batches.spec.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { matrix } from "@azure-tools/test-utils-vitest";
+import { describe, beforeEach, it } from "vitest";
+import OpenAI, { AzureOpenAI, toFile } from "openai";
+import { createClient } from "../utils/createClient.js";
+import { APIVersion, DeploymentInfo, getDeployments, withDeployments } from "../utils/utils.js";
+import { assertBatch, assertNonEmptyArray } from "../utils/asserts.js";
+import { delay } from "@azure-tools/test-recorder";
```

Comment:
are we importing the recorder anywhere else? If not, I would rather import from core-util instead.

Reply:
We can update the import. The test suite uses "assertEnvironmentVariable" imported from test-recorder. Some other recorder functions in the test suite are not removed in case we will use them again, but I can delete them.

Reply:
ah I see, we can address that in another PR.